From b8981f88d9727d439655a5edec62bdf83da166cd Mon Sep 17 00:00:00 2001
From: Roman Rakus <rrakus@redhat.com>
Date: Mon, 14 May 2012 12:58:01 +0200
Subject: [PATCH] Change `Failed' to `Unable'.

Failed word make you feel that something went in error, but this failed
was meant as warning message. Therefor changing failed to unable.

Signed-off-by: Roman Rakus <rrakus@redhat.com>
---
 src/system-config-kdump.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/system-config-kdump.py b/src/system-config-kdump.py
index f68a0c6..3976a07 100755
--- a/src/system-config-kdump.py
+++ b/src/system-config-kdump.py
@@ -559,7 +559,7 @@ class MainWindow:
         total_mem = int(total_mem + 0.99999999)
         if total_mem == 0:
             dialogs.show_message(
-                _("Failed to detect total system memory from /proc/iomem. "
+                _("Unable to detect total system memory from /proc/iomem. "
                     "Total system memory will not be accurate."),
                 _("system-config-kdump: Memory error"),
                 parent = self.toplevel)
-- 
1.7.10.1

